packages/tx/src/eip2930Transaction.ts

```diff
@@ -137,7 +138,11 @@ export default class AccessListEIP2930Transaction extends BaseTransaction<Access
    * varying data types.
    */
   public constructor(txData: AccessListEIP2930TxData, opts: TxOptions = {}) {
-    const { chainId, accessList } = txData
+    // Have to split up the const/let part to make eslint happy
+    const { chainId } = txData
+    let { accessList } = txData
+
+    accessList = accessList ?? emptyAccessList
```

Comment:
This is not necessary since in all cases default values are handled in the code below.

Reply:
Yep.